Sources/Extensions/String+OneLogin.swift

```diff
@@ -9,8 +9,10 @@ extension String {
     static let returningUser = "returningUser"
     static let returnToIntroScreen = "returnToIntroScreen"
     
+    // Wallet
+    static let hasAccessedWalletPreviously = "hasAccessedWalletPreviously"
```

Comment:
I'm not a fan of these as string extensions, can we introduce an enum for this instead? 🤔